game/CoreServices/gobfadu/gobfadupolicy/gobfadupolicyframe.rpy

```diff
@@ -0,0 +1,16 @@
+#GOBFADU Policy 1.8 DDLC Edition 6/15/2018
+#Python 3.8 MOS Edition 6/12/2018
+
+label gobfadupolicyframe:
+    stop music fadeout 2.0
+#    scene black
+#    with dissolve_scene_full
+    if renpy.exists("../game/Frameworks/UserNotifications.rpyc"):
+        call gobfaduframeverify
```

Comment:
rewrite definitions to use snake_case instead

Reply:
@GanstaKingofSA You may want to take note of this for all future GOBFADU iterations